WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderPostCardActionsHandler.kt

```diff
@@ -141,8 +141,44 @@ class ReaderPostCardActionsHandler @Inject constructor(
         }
     }
 
+    suspend fun handleFollowRecommendedSiteClicked(blogId: Long, feedId: Long?, isAskingToFollow: Boolean) {
+        val param = ReaderSiteFollowUseCase.Param.RecommendedSite(
+                blogId = blogId,
+                feedId = feedId ?: 0,
+                isAskingToFollow = isAskingToFollow
+        )
+        followUseCase.toggleFollow(param).collect {
+            when (it) {
+                is FollowSiteState.Failed.NoNetwork -> {
+                    _snackbarEvents.postValue(
+                            Event(SnackbarMessageHolder((UiStringRes(R.string.error_network_connection))))
+                    )
+                }
+                is FollowSiteState.Failed.RequestFailed -> {
+                    _snackbarEvents.postValue(
+                            Event(SnackbarMessageHolder((UiStringRes(R.string.reader_error_request_failed_title))))
+                    )
+                }
+                is FollowSiteState.Success -> Unit // Do nothing
+                is PostFollowStatusChanged -> {
+                    _followStatusUpdated.postValue(it)
+                    siteNotificationsUseCase.fetchSubscriptions()
+
+//                    if (it.showEnableNotification) {
+//                        val action = prepareEnableNotificationSnackbarAction(post.blogName, post.blogId)
+//                        action.invoke()
+//                    } else if (it.deleteNotificationSubscription) {
+//                        siteNotificationsUseCase.updateSubscription(it.blogId, DELETE)
+//                        siteNotificationsUseCase.updateNotificationEnabledForBlogInDb(it.blogId, false)
+//                    }
```

Comment:
Should we show the same info when following the Site from the "recommended_blogs" card?

Reply:
Comparing with "follow site" functionality it seems we should. Can you add it as an open question in the PR description so that we remember revisiting it?